sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java

```diff
@@ -408,6 +409,9 @@ public void canSendTypingNotificationWithResponse(HttpClient httpClient) {
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "SKIP_CHAT_READ_RECEIPT_LIVE_TESTS",
```

Comment:
nit: should we make this more generic, if we expand this to other tests as well in the future?

Reply:
Done :) thanks